_posts/2015-06-16-hard-fork-policy.md

```diff
@@ -0,0 +1,41 @@
+---
+type: posts
+layout: post
+lang: en
+category: blog
+
+title: "Bitcoin.org Hard Fork Policy"
+permalink: /en/posts/hard-fork-policy.html
+date: 2015-06-16
+---
+It appears that the recent block size debate will likely result in a
+contentious hard fork attempt.
```

Comment:
I agree with the intent, but I would prefer not to have any reference to the current block size debate. Let people who are aware make the connection, don't distract those who aren't, and don't let people who fail reading comprehension interpret this post as taking sides on the block size itself. Just remove this sentence and start the post with "Contentious hard forks are bad for Bitcoin..."